app/src/util_android.h

```diff
@@ -1127,6 +1127,12 @@ jint AttachCurrentThread(JavaVM* java_vm, JNIEnv** env);
 // App. If there is no instantiated App, returns nullptr.
 JNIEnv* GetJNIEnvFromApp();
 
+// Make sure the Java classes and methods are cached before triggering the
+// the callback. Can be slow if this is called BEFORE any Firebase App is
+// created.
+void CallAfterEnsureMethodsCached(JNIEnv* env, jobject activity,
```

Comment:
Maybe add a comment mentioning that the Unity SDK depends on this method so that it is not accidentally deleted.

Reply:
Done